Helper/Data.php

```diff
@@ -580,4 +580,27 @@ public function clearDefaultApplication()
             $applicationFactory->save();
         }
     }
+
```

Comment:
Need to add comment block that defines what the function does and what parameters it takes. Follow guide from other public functions.